src/main/java/com/commercetools/sync/categories/README.md

```diff
@@ -0,0 +1,221 @@
+objects given an old category, represented by a [Category](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/categories/Category.java),
+and a new category, represented by a [CategoryDraft](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/categories/CategoryDraft.java):
+    - ChangeName
+    - ChangeSlug
+    - SetDescription
+    - ChangeParent
+    - ChangeOrderHint
+    - SetMetaTitle
+    - SetMetaKeywords
+    - SetMetaDescription
+    - category custom fields update actions:
+        - SetCustomType
+        - SetCustomField
+
+## How to use it?
+In order to use the category sync an instance of
+ [CategorySyncOptions](https://github.com/commercetools/commercetools-sync-java/blob/develop/src/main/java/com/commercetools/sync/categories/CategorySyncOptions.java) have to be injected.
+ 
+ In order to instantiate a `CategorySyncOptions`, a `ctpClient` is required:
+  #### `ctpClient` [Required]
+  Defines the configuration of the commercetools project that categories are going to be synced to. 
+  ````java
+  // instantiating a ctpClient
+  final SphereClientConfig clientConfig = SphereClientConfig.of("project-key", "client-id", "client-secret");
+  final CtpClient ctpClient = new CtpClient(clientConfig);
+  
+  // instantiating a CategorySyncOptions
+  final CategorySyncOptions categorySyncOptions = CategorySyncOptionsBuilder.of(ctpClient).build();
+  ````
+  
+  The category sync can then do any of the following:
+  ##### sync
+  Used to sync a list of [JVM-SDK](https://github.com/commercetools/commercetools-jvm-sdk) 
+  [Category](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/categories/Category.java)
+  objects.
+  ````java
+  // instantiating a category sync
+  final CategorySync categorySync = new CategorySync(categorySyncOptions);
+  
+  // execute the sync on your list of categories
+  categorySync.sync(categories);
+  ````
+  
+  ##### syncDrafts
+  Used to sync a list of [JVM-SDK](https://github.com/commercetools/commercetools-jvm-sdk) 
+  [CategoryDraft](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/categories/CategoryDraft.java) 
+  objects.
+  ````java
+  // instantiating a category sync
+  final CategorySync categorySync = new CategorySync(categorySyncOptions);
+  
+  // execute the sync on your list of categories
+  categorySync.syncDrafts(categoryDrafts);
+  ````
+  ##### getStatistics
+  Used to get an object  containing all the stats of the sync process; which includes a report message, the total number
+  of updated, created, failed, processed categories and the processing time of the sync in different time units and in a
+  human readable format.
+  ````java
+  categorySync.syncDrafts(categoryDrafts);
```

Comment:
yes it would be affected by both runs, because it's the same instance of `statistics`. This is actually good, because if the user wants to call `sync` concurrently on multiple batches. All the statistics would be collected together.